edx_dl/edx_dl.py

```diff
@@ -189,7 +189,9 @@ def get_available_sections(url, headers):
     return sections
 
 
-def edx_get_subtitle(url, headers, get_page_contents=get_page_contents, get_page_contents_as_json=get_page_contents_as_json):
+def edx_get_subtitle(url, headers,
+                     get_page_contents=get_page_contents,
+                     get_page_contents_as_json=get_page_contents_as_json):
```

Comment:
This is OK and it improves the code, as it separates the arguments in different lines.